docs/CONTRIBUTING.md

```diff
@@ -54,7 +54,7 @@ start work on an Issue, please leave a comment so others know that it's being wo
 
 ### Tests
 
-All code changes must include tests. D2 mostly has functional tests, and uses
+All code changes must include tests. D2 mostly has functional tests (see `e2etests` directory), and uses
```

Comment:
this is not where most functional tests are. there's far more in compiler/oracle etc